dashboard/test/ui/browsers.json

```diff
@@ -2,68 +2,47 @@
 
 [
   {
-    "name": "Chrome72Win7",
-    "browserName": "chrome",
-    "version": "72",
-    "platform": "Windows 7",
-    "screenResolution": "1280x1024"
-  },
-  {
-    "name": "ChromeLatestWin7",
+    "name": "Chrome",
     "browserName": "chrome",
     "version": "latest",
     "platform": "Windows 7",
     "screenResolution": "1280x1024",
     "extendedDebugging": true
   },
   {
-    "name": "Chrome44Win7",
-    "browserName": "chrome",
-    "version": "44.0",
-    "platform": "Windows 7"
-  },
-  {
-    "name": "SafariYosemite",
+    "name": "Safari",
     "browserName": "Safari",
-    "platform": "OS X 10.10",
-    "version": "8.0"
+    "platform": "OS X 10.11",
+    "version": "10.0"
   },
   {
-    "name": "IE11Win10",
+    "name": "IE11",
     "browserName": "Internet Explorer",
     "version": "11.0",
     "platform": "Windows 10"
   },
   {
-    "name": "FirefoxLatest-2Yosemite",
-    "browserName": "firefox",
-    "version": "latest-2",
-    "platform": "OS X 10.10"
-  },
-  {
-    "name": "Firefox45Win7",
+    "name": "Firefox",
     "browserName": "firefox",
     "version": "45.0",
```

Comment:
Firefox 45 uses the legacy `FirefoxDriver` running on Selenium 2.53.1, while newer versions of Firefox use a new Firefox driver `geckodriver` which is incompatible in some ways, causing many of our current tests to fail. We need to resolve the various `geckodriver` failures in our UI tests before we can update to newer versions of Firefox.